clients/admin-ui/src/features/system/AddMultipleSystems.tsx

```diff
@@ -0,0 +1,249 @@
+/* eslint-disable react/no-unstable-nested-components */
+import {
+  Badge,
+  Button,
+  Flex,
+  Spinner,
+  Tooltip,
+  useDisclosure,
+  useToast,
+} from "@fidesui/react";
+import {
+  createColumnHelper,
+  getCoreRowModel,
+  getFilteredRowModel,
+  getPaginationRowModel,
+  getSortedRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
+import { useRouter } from "next/router";
+import { useMemo, useState } from "react";
+
+import { useAppSelector } from "~/app/hooks";
+import ConfirmationModal from "~/features/common/ConfirmationModal";
+import { useFeatures } from "~/features/common/features";
+import {
+  DefaultCell,
+  DefaultHeaderCell,
+  FidesTableV2,
+  GlobalFilterV2,
+  IndeterminateCheckboxCell,
+  PaginationBar,
+  RowSelectionBar,
+  TableActionBar,
+  TableSkeletonLoader,
+} from "~/features/common/tablev2";
+import { successToastParams } from "~/features/common/toast";
+import {
+  DictSystems,
+  selectAllDictSystems,
+  useGetAllCreatedSystemsQuery,
+  usePostCreatedSystemsMutation,
+} from "~/features/plus/plus.slice";
+
+export const VendorSourceCell = ({ value }: { value: string }) => (
+  <Flex alignItems="center" height="100%">
+    <Badge>{value.split(".")[0] === "gvl" ? "GVL" : "AC"}</Badge>
+  </Flex>
+);
+
+type MultipleSystemTable = DictSystems;
+
+const columnHelper = createColumnHelper<MultipleSystemTable>();
```

Comment:
This is a new helper function that react table provides. It makes it to typing new column definitions is really easy. It handles getting all of the types setup correctly.

Reply:
awesome!